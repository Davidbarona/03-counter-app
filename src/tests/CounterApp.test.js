import React from 'react'
import { render } from "@testing-library/react"
import CounterApp from '../CounterApp'
import {shallow} from 'enzyme'
import '@testing-library/jest-dom'

describe('testing on counterApp Component',()=>{

    test('should render CounterApp component', () => {
      const wrapper = shallow(<CounterApp />)
      expect(wrapper).toMatchSnapshot();
    });

    test('should render the value by default of  100 ', () => {
       
        const wrapper = shallow(<CounterApp value={100}/>)
        const numberDefault = wrapper.find('p').text().trim()

        expect(numberDefault).toBe("100")
    });
    


})