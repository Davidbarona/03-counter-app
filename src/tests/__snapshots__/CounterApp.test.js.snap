// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing on counterApp Component should render CounterApp component 1`] = `
<Fragment>
  <h1>
    Counter App
  </h1>
  <p>
    10
  </p>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</Fragment>
`;
